bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -44,173 +46,139 @@ __version__ = pycbc.version.git_verbose_msg
+    hdf_file: str
+        File path
     ifos: list
-    trig_table: glue.ligolw.lsctables.MultiInspiralTable, optional
+    snr_threshold: float
+        NewSNR threshold
+    bank_file: h5py.File object
+    trig_data: h5py.File object
     background_bestnrs: numpy.array, optional
         Used to compute FAP of quiet injections.
 
     Returns
     -------
-    inj_data: dict of parameter numpy.array pairs
-        Found or missed injection parameter dictionary.
+    data: tuple of dictionaries
+        Found and missed injection parameter dictionaries.
     """
 
-    inj_data = {}
-
-    # Local copies of variables entering the BestNR definition
-    chisq_index = opts.chisq_index
-    chisq_nhigh = opts.chisq_nhigh
-    null_thresh = list(map(float, opts.null_snr_threshold.split(',')))
-    snr_thresh = opts.snr_threshold
-    sngl_snr_thresh = opts.sngl_snr_threshold
-    new_snr_thresh = opts.newsnr_threshold
-    null_grad_thresh = opts.null_grad_thresh
-    null_grad_val = opts.null_grad_val
-
-    # Get inj params
-    inj_params = ['mchirp', 'mass1', 'mass2', 'distance', 'inclination',
-                  'spin1x', 'spin1y', 'spin1z', 'spin2x', 'spin2y', 'spin2z']
+    inj_data = h5py.File(hdf_file, 'r')
+    inj_params = ['mass1', 'mass2', 'distance', 'inclination', 'ra', 'dec',
+                  'polarization', 'spin1x', 'spin1y', 'spin1z', 'spin2x',
+                  'spin2y', 'spin2z', 'tc']
+    found_data = {}
+    # Missed injections (ones not recovered at all)
+    missed_data = {}
+    logging.info('Loading injections...')
+
+    # Load injections parameters
     for param in inj_params:
-        inj_data[param] = get_column(inj_table, param)
-    inj_data['ra'] = get_column(inj_table, 'longitude')
-    inj_data['dec'] = get_column(inj_table, 'latitude')
-    inj_data['time'] = get_column(inj_table, 'geocent_end_time') \
-                + 1e-9 * get_column(inj_table, 'geocent_end_time_ns')
+        missed_data[param] = inj_data['missed/%s' % param][...]
+        found_data[param] = inj_data['found/%s' % param][...]
+
+    # Calculate injections mchirp
+    missed_data['mchirp'] = mchirp_from_mass1_mass2(missed_data['mass1'],
+                                                    missed_data['mass2'])
+    found_data['mchirp'] = mchirp_from_mass1_mass2(found_data['mass1'],
+                                                   found_data['mass2'])
+
+    # Calculate effective distance for the ifos
+    found_data = additional_injection_data(found_data, ifos)
+    missed_data = additional_injection_data(missed_data, ifos)
+
+    # Get recovered parameters and statistic values for the found injections
+    # Recovered parameters
+    for param in ['mass1', 'mass2', 'spin1z', 'spin2z']:
+        found_data['rec_%s' % param] = np.array(bank_file[param])[inj_data['network/template_id']]
+
+    found_data['end_time_gc'] = \
+        inj_data['network/end_time_gc'][...]  # needed to use bestNR
```

Comment:
- I'm not sure what you mean with "needed to use bestNR" here: remove the comment if it doesn't apply.
- `found_data['end_time_gc']` is filled up but we never use it.  Consider an additional output column with `tc-end_time_gc`.